---
bool[16] status
string message